content/terms.md

```diff
@@ -53,48 +53,50 @@ feedback](mailto:support@illumitacit.com).
-   plan that is billed on January 1st of every year and you decide to purchase more word capacity on April 1st, you will
-   pay for those words for the rest of the year at a pro rated amount (3/4 the full amount), and then will be billed
-   annually every January 1st for the full cost.
-
-6. You can lower the available word capacity in your plan at any time within your account settings. You will
-   keep the current word capacity until the end of the billing cycle, at which point the new amount will be reflected.
-   IllumiTacit does not issue refunds for changes in word capacity. Lowering the number of words that are available in
-   your plan may cause loss of capacity in your account which may make your existing usage untenable. IllumiTacit does
-   not accept any liability for such loss.
-
-7. Our payment process is conducted by our Merchant of Record ([Paddle](https://www.paddle.com/)). All fees are
+5. Usage is tracked by the number of words that are generated by our engine. Words are counted as contiguous
+   non-whitespace characters in the text. For languages that are not space-delimited, such as Chinese and Japanese,
+   words are counted as non-whitespace, non-punctuation characters in the text.
+
+6. The paid plan includes a certain number of words that are replenished into your account balance every billing cycle,
+   as described in [the pricing table](https://illumitacit.com/#pricing). You will not be able to generate content if
+   your account balance drops to, or below, 0. We will not charge for a negative balance on your account. However, a
+   negative balance will be counted when replenishing your account. Words in your account balance will never expire,
+   unless you delete your account. Words hold no monetary value and are non-refundable. In other words, you may not
+   exchange words in your account balance for monetary transactions, or for a refund, even after cancellation.
+
+7. You can purchase additional word capacity in your plan at any time within your account settings. Additional words are
+   charged on a one time basis and will immediately replenish your account. You can also configure auto-reload on your
+   account. When auto-reload is enabled, your account will automatically be charged for the number of words you have
+   configured when your balance drops below the configured threshold.
+
+8. Our payment process is conducted by our Merchant of Record ([Paddle](https://www.paddle.com/)). All fees are
    inclusive of all taxes, levies, or duties imposed by taxing authorities. Paddle will collect those taxes on behalf of
    the taxing authorities and remit those taxes to them. All charges will originate from Paddle and will show as
-   `PADDLE.NET* ILLUMITACIT` on your bank/card statement. See [Paddle's Terms of
+   `PADDLE.NET* ILLUMITACI` on your bank/card statement. See [Paddle's Terms of
```

Comment:
```suggestion
   `PADDLE.NET* ILLUMITACIT` on your bank/card statement. See [Paddle's Terms of
```

Reply:
Was this intentional?

Reply:
Yea this was intentional. I was testing this new model on the sandbox and then noticed that the receipts has it cut off. See bottom of the message here:

<img width="621" alt="Screenshot 2023-08-18 at 5 43 36 AM" src="https://github.com/illumitacit/policies/assets/430092/8d720232-eb68-4194-8c38-72c5cb6e9be8">